projects/core/directives/dropdown/dropdown.component.ts

```diff
@@ -67,7 +67,11 @@ export class TuiDropdownComponent implements OnInit {
             takeUntilDestroyed(),
         )
         .subscribe(([top, left]) => {
-            this.update(top, left);
+            if (this.directive.el.isConnected) {
+                this.update(top, left);
+            } else {
+                this.directive.toggle(false);
+            }
```

Comment:
Is this how it should be?

<img width="1284" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/da0c2823-4b0f-40f7-84f4-0054e286e5ac">

Reply:
Yes, approach from the screenshot is intended for hints, because the hints don't close immediately, but with a delay. The dropdown does not have this problem